internal/common/shared/gifts.go

```diff
@@ -0,0 +1,29 @@
+package shared
+
+// GiftDict is a dictonary of gift
+type GiftDict = map[ClientID]int
+
+// AcceptReason is a just a logical wrapper on the int, a normal int declaration could easily have been used.
+type AcceptReason int
+
+const (
```

Comment:
![image](https://user-images.githubusercontent.com/33488131/102896623-99083a80-445e-11eb-8da7-a81cc2edf11b.png)

sad but this should shut linter up

Reply:
I mean these consts are quite selfexplanatory, can add another comment to add their intended use

Reply:
These enums are quite self-explanatory but I can add another definition to satisfy the go gods

Reply:
hail rob pike and ken thompson